examples/src/main/java/tachyon/examples/BasicRawTableOperations.java

```diff
@@ -49,10 +50,10 @@
   private long mId;
 
   public BasicRawTableOperations(TachyonURI masterAddress, TachyonURI tablePath,
-      WriteType writeType) {
+      TachyonStorageType writeType) {
     mMasterAddress = masterAddress;
     mTablePath = tablePath;
-    mWriteType = writeType;
+    mWriteType = writeType.isStore() ? WriteType.MUST_CACHE : WriteType.THROUGH;
```

Comment:
This looks like it is translating the new types to the old types. Why isn't the new types (and new client) used all along?

Also, shouldn't the test then take both TachyonStorageType and UnderStorageType to accurately represent the old WriteTypes?